apps/modernization-api/src/main/java/gov/cdc/nbs/search/WildCards.java

```diff
@@ -4,20 +4,27 @@ public class WildCards {
 
   public static String startsWith(final String value) {
     return isValid(value)
-        ? value.toLowerCase().trim() + "*"
+        ? escape(value.toLowerCase().trim()) + "*"
         : null;
   }
 
   public static String contains(final String value) {
     return isValid(value)
-        ? "*" + value.toLowerCase().trim() + "*"
+        ? "*" + escape(value.toLowerCase().trim()) + "*"
         : null;
   }
 
   private static boolean isValid(final String value) {
     return value != null && !value.isEmpty();
   }
 
+  private static String escape(final String value) {
+    if (value == null) {
+      return null;
+    }
```

Comment:
suggestion:  in it's current usage `value` will never be null and this check can be skipped.  This will also remove a branch from the code coverage calculation

Reply:
I removed it